### PR TITLE
[inductor] Small fixes for huggingface models

### PR DESCRIPTION
**Pull request by jansel**

### CODE REVIEW

**Review by ngimel — APPROVED** [review]

File: torchinductor/sizevars.py [hunk 1]
```diff
@@ -11,6 +12,8 @@
 
 from .virtualized import V
 
+log = logging.getLogger(__name__)
```
Comment (desertfire) [inline torchinductor/sizevars.py:15]: Used logging and removed for this PR?

Comment (jansel) [reply torchinductor/sizevars.py:15]: Yeah, I think it is fine to leave this line for the future though.

**Review by desertfire — APPROVED** [review]

File: torchinductor/lowering.py [hunk 2]
```diff
@@ -271,12 +271,26 @@ def _to_copy(
...
+    layout=None,
+):
     assert not memory_format, "TODO"
+    assert layout in (None, torch.strided)
     if isinstance(device_or_dtype, torch.dtype):
         return to_dtype(x, device_or_dtype)
     if isinstance(device_or_dtype, torch.device):
         return to_device(x, device_or_dtype)
+    if device is not None:
+        return to_device(x, device)
```
Comment (ngimel) [inline torchinductor/lowering.py:291]: this will be wrong if both device and dtype is specified?

Comment (jansel) [reply torchinductor/lowering.py:291]: Good catch, will update.

**Pull request merged**